backend/tex/aux_parser.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import json
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--file', help='filepath to input aux')
+args = parser.parse_args()
+
+f = open(args.file, 'r')
+s = '\\newlabel'
+
+listOfLabels = []
+
+def getArg(s, i):
+    end = s[i+1:].find("}")
```

Comment:
`string.find` in Python accepts optional arguments indicating the substring. I suggest using them because it will eliminate slippery arithmetics below